app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -260,10 +284,6 @@ def resource_scope(_options = {})
         @resource_scope ||= ForemanTasks::Task.authorized("#{action_permission}_foreman_tasks")
       end
 
-      def resource_class
-        @resource_class ||= ForemanTasks::Task
-      end
```

Comment:
Is this really not needed anymore?

Reply:
Not sure how/why it was removed, but I added it back!